app/controllers/user.js

```diff
@@ -19,21 +19,26 @@ exports.newUser = (req, res, next) => {
 
   const valMsgs = validation.validateUser(newUser);
   if (valMsgs.length > 0) {
-    next(errors.defaultError(valMsgs));
+    next(errors.invalidUser(valMsgs));
   } else {
     return bcrypt
       .hash(newUser.password, saltRounds)
       .then(hash => {
         newUser.password = hash;
 
-        return User.createNewUser(newUser)
+        return User.createModel(newUser)
           .then(user => {
-            res.status(201).end();
+            res.statusMessage = `Successfully created new user. Welcome, ${newUser.firstName} ${
+              newUser.lastName
+            }!`;
+            res.status(200).end();
```

Comment:
Why you change status code 201 created per 200 and a message ? This only add test and documentation no change endpoints behavior